### PR TITLE
2060 text formatting button

### PR DESCRIPTION
**Pull request by BlaineOmega**
Addresses #2060 . ~~Removed the "isActive" check from the shortcut and used the commandStateStore to control the state of the button. There was no need for the isActive function, so I removed it.~~

### CODE REVIEW

File: src/@types/Shortcut.ts [hunk 1]
```diff
@@ -48,9 +48,6 @@ interface Shortcut {
   /** A readable, internal unique id. */
   id: ShortcutId
 
-  /** A function that returns true if the shortcut should be highlighted in the Toolbar. */
-  isActive?: (getState: () => State) => boolean
-
```
Comment (raineorshine) [inline src/@types/Shortcut.ts:53]: `isActive` is used by other (non-formatting) shortcuts, so we cannot remove it.

e.g. Try toggling sort:

https://github.com/cybersemics/em/blob/a33035e2076d82d8114459d662b0508ca353d686/src/shortcuts/toggleSort.ts#L38-L45

File: src/components/ToolbarButton.tsx [hunk 2]
```diff
@@ -49,17 +49,18 @@ const ToolbarButtonComponent: FC<DraggableToolbarButtonProps> = ({
...
     throw new Error('The svg property is required to render a shortcut in the Toolbar. ' + shortcutId)
   }
 
   const commandState = commandStateStore.useSelector(state => state[shortcutId])
+
   const isDraggingAny = useSelector(state => !!state.dragShortcut)
   const dragShortcutZone = useSelector(state => state.dragShortcutZone)
-  const isButtonActive =
-    useSelector(state => (customize ? selected : !isActive || isActive(() => state))) || (!customize && commandState)
+  const isButtonActive = useSelector(state => (customize ? selected : commandState))
```
Comment (raineorshine) [inline src/components/ToolbarButton.tsx:62]: You have deleted the `customize` logic. When in toolbar customization mode, the active state is set to `selected`.

You can test this out by going to Settings → Customize Toolbar and selecting a toolbar button.

**Review by raineorshine — CHANGES_REQUESTED** [review]

File: src/components/ToolbarButton.tsx (hunk 2 above)
Comment (BlaineOmega) [reply src/components/ToolbarButton.tsx]: I fixed this issue. Works for both the customize and the regular tool bar. There logic is much simpler :)

File: src/@types/Shortcut.ts (hunk 1 above)
Comment (BlaineOmega) [reply src/@types/Shortcut.ts]: I added this back to the interface.

**Review by raineorshine — APPROVED** [review]

**Comment by raineorshine** [comment]
Looks good, thank you!

**Pull request merged**